extension/persistence/relational-jdbc/build.gradle.kts

```diff
@@ -32,6 +32,7 @@ dependencies {
   compileOnly(libs.jakarta.inject.api)
 
   implementation(libs.smallrye.common.annotation) // @Identifier
+  compileOnly(libs.smallrye.config.core)
```

Comment:
Why do we need this?

Reply:
for the ConfigMap we have this is in the persistence code